### PR TITLE
Development add support for iot repository packages

### PR DESCRIPTION
**Pull request by joaoopereira**

### CODE REVIEW

File: cmf-cli/Handlers/PackageType/IoTRepositoryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IoTRepositoryHandler" /> class.
+        /// </summary>
+        /// <param name="cmfPackage">The CMF package.</param>
+        public IoTRepositoryHandler(CmfPackage cmfPackage) : base(cmfPackage)
+        {
+            cmfPackage.SetDefaultValues
+            (
+                //targetDirectory:
```
Comment (joaoopereira) [inline cmf-cli/Handlers/PackageType/IoTRepositoryHandler.cs:31]: remove commented code

File: cmf-cli/Handlers/PackageType/IoTRepositoryHandler.cs [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        {
+            FileSystemUtilities.CopyInstallDependenciesFiles(packageOutputDir, PackageType.IoT, this.fileSystem);
+        }
+
+        /// <summary>
+        /// Packs the specified package output dir.
+        /// </summary>
+        /// <param name="packageOutputDir">The package output dir.</param>
+        /// <param name="outputDir">The output dir.</param>
+        public override void Pack(IDirectoryInfo packageOutputDir, IDirectoryInfo outputDir)
```
Comment (joaoopereira) [inline cmf-cli/Handlers/PackageType/IoTRepositoryHandler.cs:110]: duplicated code

File: cmf-cli/Handlers/PackageType/IoTRepositoryHandler.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using Cmf.Common.Cli.Commands.restore;
+using Newtonsoft.Json.Serialization;
+
+namespace Cmf.Common.Cli.Handlers
+{
+    /// <summary>
+    ///
```
Comment (m-s-) [inline cmf-cli/Handlers/PackageType/IoTRepositoryHandler.cs:15]: please fill in the comments...

File: cmf-cli/Objects/Step.cs [hunk 4]
```diff
@@ -80,15 +97,41 @@ public class Step : IEquatable<Step>
...
+        [JsonProperty(Order = 11)]
+        public string TargetFile { get; set; }
+
+        /// <summary>
+        /// Value
+        /// </summary>
+        /// <value>
+        /// Value.
+        /// </value>
+        [JsonProperty(Order = 12)]
```
Comment (m-s-) [inline cmf-cli/Objects/Step.cs:123]: what are these 2 for?

**Review by m-s- — CHANGES_REQUESTED** [review]

**Comment by m-s-** [comment]
This was superseeded by new DF steps available in 8.3.5.

**Pull request closed without merging**